src/shared/condition.c

```diff
@@ -578,8 +579,51 @@ static int condition_test_firmware_devicetree_compatible(const char *dtcarg) {
         return strv_contains(dtcompatlist, dtcarg);
 }
 
+static int condition_test_firmware_smbios_field(const char *expression) {
+        int r;
+        _cleanup_free_ char *field = NULL, *expected_value = NULL, *actual_value = NULL;
+        OrderOperator operator;
+
+        assert(expression);
+
+        /* Parse SMBIOS field */
+        r = extract_first_word(&expression, &field, "!<=>", EXTRACT_RETAIN_SEPARATORS);
```

Comment:
given the `"!<=>"` is now used at at least two places, please add a `#define` macro for it

Reply:
This is not the case anymore until `condition_test_osrelease()` is extended with the new `fnmatch()` operator.